src/components/desktop/Header.js

```diff
@@ -27,12 +30,37 @@ const StyledHeader = styled.header`
       li {
         display: inline-grid;
         place-items: center;
-        font-weight: bold;
         width: 110px;
         margin: 0;
         height: 100%;
-        color: ${({ theme }) => theme.background};
+        color: ${({ theme }) => theme.text};
         cursor: pointer;
+        position: relative;
+
+        a {
+          text-decoration: none;
+          color: ${({ theme }) => theme.text};
```

Comment:
Should the color here be a darker gray? Looks lighter on the deployment than the Figma. Slightly hard to see. @SirajChokshi 

Also text size seems too small for the header, but not sure.

Reply:
Yes the header's height currently looks too large relative to the text size